### PR TITLE
Make support portal easier to find

### PR DESCRIPTION
**Pull request by Neves-P**
Adds links to the [support portal](https://gitlab.com/eessi/support) in the [contact](https://www.eessi.io/docs/contact/) and [adding software overview pages](https://www.eessi.io/docs/adding_software/overview/). Also changes mention of https://www.eessi-hpc.org/ in contact page to eessi.io

### CODE REVIEW

**Review by boegel — APPROVED** [review]

File: docs/adding_software/overview.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 * [Building software *(for maintainers)*](building_software.md)
 * [Debugging failed builds *(for contributors + maintainers)*](debugging_failed_builds.md)
 * [Deploying software *(for maintainers)*](deploying_software.md)
+* [Support portal *(for contributors + maintainers)*](https://gitlab.com/eessi/support)
```
Comment (boegel) [inline docs/adding_software/overview.md:8]: This should point to https://www.eessi.io/docs/support instead

Also, maybe it shouldn't be a bullet point, and a small section at the bottom, like:

```suggestion

If you need help with adding software to EESSI, please [open a support request](support.md).

File: docs/contact.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 For more information:
 
-* visit our website [https://www.eessi-hpc.org](https://www.eessi-hpc.org)
+* visit our website [https://www.eessi.io](https://www.eessi.io)
 * consult our documentation at [https://eessi.github.io](https://eessi.github.io/docs)
+* ask for help at our [support portal](https://gitlab.com/eessi/support)
```
Comment (boegel) [inline docs/contact.md:7]: also here, I would point to the support section of our docs, which provides more info:

```suggestion
ask for help at our [support portal](support.md)
```

**Review by boegel — CHANGES_REQUESTED** [review]

File: docs/contact.md (hunk 2 above)
Comment (Neves-P) [reply docs/contact.md]: Agreed, but on this page I left it as a bullet point, let me know if you'd prefer otherwise

**Comment by Neves-P** [comment]
Thanks for the review, @boegel! I've addressed both points now

**Comment by Neves-P** [comment]
Build was failing on a broken link. Fixed the link and it passes now.

**Review by boegel — APPROVED** [review]

**Pull request merged**